server/@types/forms/index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare module 'forms' {
   export interface CreateGoalForm {
-    readonly prisonNumber: string
+    prisonNumber: string
```

Comment:
I've changed this from a readonly property; and have added it as a hidden field on the HTML form (see the change to the nunjucks view below)
It makes the controller much simpler in respect of populating the form in the session